index.js

```diff
@@ -10,19 +10,22 @@ const createRobot = require('./lib/robot')
 const createServer = require('./lib/server')
 const serializers = require('./lib/serializers')
 
-module.exports = (options = {}) => {
-  const cache = cacheManager.caching({
-    store: 'memory',
-    ttl: 60 * 60 // 1 hour
-  })
+const cache = cacheManager.caching({
+  store: 'memory',
+  ttl: 60 * 60 // 1 hour
+})
 
-  const logger = bunyan.createLogger({
-    name: 'PRobot',
-    level: process.env.LOG_LEVEL || 'debug',
-    stream: bunyanFormat({outputMode: process.env.LOG_FORMAT || 'short'}),
-    serializers
-  })
+const logger = bunyan.createLogger({
+  name: 'PRobot',
```

Comment:
nit: `Probot`

Reply:
I'll leave this and we can updated it in #230